front/pages/api/v1/w/[wId]/assistant/conversations/[cId]/messages/[mId]/events.ts

```diff
@@ -161,6 +161,10 @@ async function handler(
         res.flush();
       }
 
```

Comment:
Did you assess the impact of this diff on the main app?

Reply:
yes, it has no impact as the stream is closed before receiving the message, but it's actually not needed anymore in extension neither. will clean up.

Reply:
Roger that

Reply:
Btw, the main app does not use this endpoint, so it's clearly not affected. Receiving `done` only happens in the case we don't get messages in 1 min (redis timeout). This was done for conversation streaming (waiting for new messages) but not for message streaming (waiting for new message block) - as message streaming in never stuck so long (and we close the stream as soon as the message is completed)
It would be more consistent and safer to send a `done` event at the end all sse endpoints, but that would change the public endpoint return and is not required here, so rather leaving as is.

Reply:
Thanks for the analysis. Let's fix if relevant 👍